manifests/z/Ziad/TCG/6.8/Ziad.TCG.locale.en-US.yaml

```diff
@@ -0,0 +1,37 @@
+# Created with YamlCreate.ps1 v2.2.8 $debug=NVS1.CRLF.5-1-19041-3031.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Ziad.TCG
+PackageVersion: "6.8"
+PackageLocale: en-US
+Publisher: Ziad (Mr.X) Software
+PublisherUrl: https://insertx2k.github.io/mrx
+PublisherSupportUrl: https://insertx2k.github.io/mrx
+PrivacyUrl: https://insertx2k.github.io/temp_cleaner_gui/privacy.html
+Author: Ziad (Mr.X) Software
+PackageName: Temp_Cleaner GUI
+PackageUrl: https://insertx2k.github.io/temp_cleaner_gui
+License: GPL
```

Comment:
```suggestion
License: GPL-2.0
```

Reply:
The license version is GPL 2+ (which means GNU General Public License v2.0 or newer), that's why I put this as `GPL` instead of `GPL-2.0`

Reply:
In that case we can set it to `GPL-2.0-or-later`

Reply:
I used the interactive `YamlCreate.ps1`, I didn't know that this was an available option as it wasn't listed there.